crates/rspack_plugin_javascript/src/visitors/swc_visitor/react.rs

```diff
@@ -126,7 +126,7 @@ fn create_react_refresh_runtime_stmts(unresolved_mark: Mark) -> Vec<Stmt> {
                     left: BinExpr {
                       span: DUMMY_SP,
                       op: BinaryOp::Add,
-                      left: member_expr!(DUMMY_SP, __webpack_module__),
+                      left: member_expr!(DUMMY_SP, __webpack_module__.id),
```

Comment:
Using plain AST here makes code hard to review... Maybe we can use macro `quote` to optimize review experience for the team.

Reply:
needs add unresolved_mark, not sure whether `quote` can do that

Reply:
> needs add unresolved_mark, not sure whether `quote` can do that

You bet! Please refer to https://github.com/swc-project/swc/blob/e4f9f734ad1f92c6f05e8cb4c2d799679cca9f79/crates/swc_core/src/quote.rs#L13 for more details.

Reply:
This is cool! let me update this